test/ByteCode/rlexe.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<regress-exe>
+  <test>
+    <default>
+      <tags>exclude_dynapogo,exclude_amd64,exclude_arm,require_backend</tags>
+      <compile-flags>-GenerateLibraryByteCodeHeader</compile-flags>
```

Comment:
We used to use scripts in SD but never ported it over to git. There essentially doesn't need to be a script; it was doing what RL does. The only real change is that instead of writing to a file I write to stdout.